xml/System.Windows.Forms/ListViewItemStateImageIndexConverter.xml

```diff
@@ -54,9 +54,9 @@
         <Parameter Name="context" Type="System.ComponentModel.ITypeDescriptorContext" />
       </Parameters>
       <Docs>
-        <param name="context">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="context">An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context, which can be used to extract additional information about the environment this type converter is being invoked from. This parameter or properties of this parameter can be <see langword="null" />.</param>
```

Comment:
```suggestion
        <param name="context">An object that provides a format context, which can be used to extract additional information about the environment this type converter is being invoked from. This parameter or its properties can be <see langword="null" />.</param>
```
It's actually the argument that can be null, though we use "this parameter can be null" everywhere. But properties of the parameter being null is more of a stretch...